webui/react/src/pages/ClusterV1.tsx

```diff
@@ -1,58 +0,0 @@
-import React, { useEffect, useMemo, useState } from 'react';
-
-import Grid from 'components/Grid';
-import Message from 'components/Message';
-import Page from 'components/Page';
-import ResourceChart from 'components/ResourceChart';
-import Spinner from 'components/Spinner';
-import Agents from 'contexts/Agents';
-import { Resource, ResourceType } from 'types';
-import { categorize } from 'utils/data';
```

Comment:
same for `categorize`, was used only here.

Reply:
🤔  this one I think we'll use and it shouldn't add to our bundle size as long as we don't import and use it anywhere. If you prefer we don't keep it I'll remove it

Reply:
totally fine to keep!